src/Rings/mpolyquo-localizations.jl

```diff
@@ -2155,4 +2155,92 @@ function inverse(phi::MPolyQuoLocalizedRingHom)
   error("computation of inverse not implemented")
 end
 
+@doc raw"""
+    minimal_generating_set(I::MPolyLocalizedIdeal)
+
+Given an ideal `I` in the localization of a quotient of a  multivariate
+polynomial ring over a field at a point, return an array containing a
+minimal set of generators of `I`. If `I` is the zero ideal an empty list
+is returned.
+
+Note: This is only available for localizations at rational points. 
+"""
+@attr Vector{<:MPolyQuoLocRingElem}function minimal_generating_set(
+    I::MPolyQuoLocalizedIdeal{<:MPolyQuoLocRing{<:Field, <:Any,
+                                          <:Any, <:Any,
```

Comment:
Out of curiosity? Did the type assertions here cause any trouble? Otherwise: Why not keep them?

Reply:
Yes and no, it did cause trouble during the my many tests and debugging runs because of my incompetence to always feed suitable objects to it. The two imposed conditions suffice for us, so I left out the others.